cmd/gitbase/command/server_test.go

```diff
@@ -1,9 +1,14 @@
 package command
 
 import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	fixtures "gopkg.in/src-d/go-git-fixtures.v3"
```

Comment:
fixtures should not be v3, but "github.com/src-d/go-git-fixtures", which is the one we're using throughout the code

Reply:
Right! I should stop relying blindly in `goimports`. Fixed.